css-ui-3/caret-color-017.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: caret-color transition</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#caret-color">
+<link rel="help" href="https://www.w3.org/TR/css3-transitions/#transition">
+<meta name="flags" content="interact animated">
+<meta name="assert" content="Test checks that caret-color can be used in a transition">
+<style>
+  textarea {
+    font-size: 3em;
+    font-weight: bold;
+    width: 10em;
+    padding: 10px;
+    background: black; color: white; /* the color of a thin object like the caret is easier to see on a black background. */
+
+    caret-color: magenta;
+    transition: caret-color 5s;
+  }
+
+  textarea:focus {
+    caret-color: lime;
+  }
+</style>
+<body>
+  <p>Test passes if, when the text area below is focused for editing, the text insertion caret color transitions from magenta to green.</p>
+  <p>The shape of the caret, and whether it flashes, are not part of the test.</p>
+  <textarea autofocus></textarea>
```

Comment:
maybe remove the autofocus attribute, so that people have to focus by themselves, and get  chance to observe the transition?